src/CMakeLists.txt

```diff
@@ -61,7 +57,6 @@ vala_init(ayatana-indicator-sound-service
     --pkg=bus-watcher
     ${VALA_DEFINE_LOMIRI_SOUND_SCHEMA}
     ${VALA_DEFINE_ACCTSERVICE_PRIVACY_SETTINGS}
-    ${VALA_DEFINE_ACCTSERVICE_SOUND_SETTINGS}
     ${VALA_DEFINE_ACCTSERVICE_SYSTEMSOUND_SETTINGS}
```

Comment:
Keep this.

Reply:
Ignoring, as we agreed to simplify this in the next PR.